src/tests/test_installations.py

```diff
@@ -15,6 +15,11 @@
 ]
 
 
+async def mock_get_custom_entry(query):
+
+    return [INSTALLATION_TABLE[0]["device_id"]]
+
+
```

Comment:
That's one problem with this crud, we can't mock it properly. This override is completely static, which is a risk